yevhenii.dolmatov/task_07/image.rb

```diff
@@ -0,0 +1,20 @@
+require 'faraday'
+
+class Image
+  def download(url)
+    response = Faraday.get(url)
+    original_extension = url.split(//).last(4).join
```

Comment:
it won't work if you would have URL like this:
https://miro.medium.com/max/4000/1*KUy_KKExZrSpBuv9XfyBgA.png?somasda=asd&location=Cherkassy

please fix to support all types of urls